packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -455,6 +445,16 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
     const SingleActivator(LogicalKeyboardKey.arrowRight, shift: true, alt: true): const DoNothingAndStopPropagationTextIntent(),
     const SingleActivator(LogicalKeyboardKey.arrowLeft, shift: true, meta: true): const DoNothingAndStopPropagationTextIntent(),
     const SingleActivator(LogicalKeyboardKey.arrowRight, shift: true, meta: true): const DoNothingAndStopPropagationTextIntent(),
+    const SingleActivator(LogicalKeyboardKey.pageUp): const DoNothingAndStopPropagationTextIntent(),
```

Comment:
I think these shortcuts should not be disabled if `kIsWeb` && `defaultTargetPlatform == TargetPlatform.macOS`. The reason why they are currently disabled for macOS is because we receive selectors from the macOS embedder that we convert to intents. This happens in https://github.com/flutter/flutter/blob/433bca5edb0bb9646174a4f23d3710b7145626f5/packages/flutter/lib/src/widgets/editable_text.dart#L4344-L4354 . The web embedder does not send these selectors iirc, so disabling these shortcuts here would result in these shortcuts not working on macOS web.

Reply:
Sorry disregard my initial comment. I had the logic for `getDisablingShortcut()` mixed up. This change should work fine for macOS native and macOS web.